dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/util/Stats.kt

```diff
@@ -25,15 +25,18 @@ import javafx.beans.property.StringProperty
 import java.time.Instant
 
 class Stats(val message: StringProperty, val loggingService: LoggingService, val dateTimeUtils: DateTimeUtils) {
-    fun updateStatistics(name: String,
-                         startTime: Instant,
+    private var lastFile: String = ""
+
+    fun updateStatistics(startTime: Instant,
                          sent: LongProperty,
                          total: LongProperty,
                          totalMessage: String,
                          toPath: String,
                          location: String,
-                         finished: Boolean) {
+                         finished: Boolean,
+                         name: String = lastFile) {
```

Comment:
is there a reason we need a default value? i.e. whats the use case where its updated and its not because a file finished?

Reply:
There are cases where we want to update statistics but didn't receive a last file name, but most of the time we will receive a last file name.

Reply:
got it